### DISCUSSION TITLE
current `function name` or `TS-object` to be shown on statusline.

### DISCUSSION THREAD

**Discussion by masriomarm** [helix-editor/helix#7813] [Ideas]
while cursor is within a function body, what are the possible ways to tell that function name?

**Comment by gabydd** [comment]
Right now there isn't really a good way except for saving a jump point then going up to look but sticky context: https://github.com/helix-editor/helix/pull/6118 would let you see which function you are in

**Reply by willhansen** [reply]
That seems a bit overkill for this tbh.  

Something like having `<file-name>:<function-name>` in the status line is probably the least intrusive way to display it.

`mif` selects the current function, so there's tree-sitter support for this somehow.  

Edit: sticky-context is the correct solution to this problem.

**Comment by zhaob1n** [comment]
still looking for this function
